projects/rero/ng-core/src/lib/record/editor/type/password-generator-type.component.ts

```diff
@@ -53,6 +53,7 @@ import { tap } from 'rxjs/operators';
 
 interface PasswordGeneratorProps extends FormlyFieldProps {
   enabledEditMode: boolean;
+  class?: string;
```

Comment:
Do you need a default value?

Reply:
No, because if edit mode is enabled, it doesn't need a class. I think it's better to set the class when you need it.